presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -77,6 +77,10 @@ Property Name                                      Description
 ``cassandra.password``                             Password used for authentication to the Cassandra cluster.
                                                    This is a global setting used for all connections, regardless
                                                    of the user who is connected to Presto.
+
+``cassandra.protocol-version``                     It is possible to override the protocol version for older Cassandra clusters.
+                                                   Default value: V3
+                                                   Alternative values: V2 or V4
```

Comment:
We should add a warning here that setting the protocol version to a version that does not match your Cassandra installation can causes problems (e.g. using V4 causes intermittent issues with 2.1.x versions of Cassandra). Maybe there's a Cassandra driver doc you can refer to about protocol versions?

Also, please use full sentences to indicate the default -- e.g., "This property defaults to ``V3``. Possible values include ``V2``, ``V3`` and ``V4``."